pkg/aws/client.go

```diff
@@ -1212,30 +1212,42 @@ func (c *awsClient) DeleteSecretInSecretsManager(secretArn string) error {
 }
 
 func (c *awsClient) GetSecurityGroupIds(vpcId string) ([]ec2types.SecurityGroup, error) {
```

Comment:
@den-rgb It would be nice to see a unit test for this. The EC2 client is mocked already in the `client_test.go` so it shouldn't be too complicated.

Reply:
done